README.md

```diff
@@ -140,6 +140,31 @@ quark -a first.apk -a second.apk -C
 
 <img src="https://i.imgur.com/ClRWOei.png"/>
 
+### Radiocontrast
+The Radiocontrast is a Quark API that can quickly generate 100% confidence rules with the Native APIs used in a specific method. The feature lets you easily see through the behavior of a method, just like radiocontrast.
+
+For example, if you want to see the behavior method in `Ahmyth` called `Lahmyth/mine/king/ahmyth/CameraManager;->startUp(I)V,` 
+here is the simplest way for usage:
```

Comment:
Since the sentence is not grammatically correct, I suggest inserting the word "Radiocontrast" between the word "for" and the word "usage".